### PR TITLE
remove custom fontawesome icons

### PR DESCRIPTION
**Pull request by UpcraftLP**
This is a bandaid fix for #225, replacing custom fontawesome icons with their standard variants, and where that is not possible (curseforge and modrinth logo, pineapple), ships raw SVG images instead.

---
See preview on Cloudflare Pages: https://preview-226.quiltmc-org.pages.dev

### CODE REVIEW

**Comment by Pyrofab** [comment]
So this keeps FontAwesome, but removes the reliance on our FA Pro account ?

**Comment by UpcraftLP** [comment]
> So this keeps FontAwesome, but removes the reliance on our FA Pro account ?

correct

File: src/components/parts/team/TeamMember.astro [hunk 1]
```diff
@@ -11,6 +15,13 @@ const { member, isLeader } = Astro.props as Props;
 
 // this is the "N/A" text, for anyone wondering
 const na = t("team:n-a");
+
+let customIcons: any[any] = {
```
Comment (Pyrofab) [inline src/components/parts/team/TeamMember.astro:19]: Shouldn't this be a `Record<string, string>` ?

**Review by Pyrofab — APPROVED** [review]
Epic

**Pull request merged**